wsmaster/che-core-api-factory-bitbucket-server/src/main/java/org/eclipse/che/api/factory/server/bitbucket/BitbucketUrl.java

```diff
@@ -148,10 +145,11 @@ public String rawFileLocation(String fileName) {
             .add(repository)
             .add("raw")
             .add(fileName);
+    String resultUrl = joiner.toString();
```

Comment:
can you explain why we need this change?

Reply:
because query should not be joined via `/`